lib/aws/awsconfigfile/awsconfigfile.go

```diff
@@ -0,0 +1,117 @@
+// Teleport
+// Copyright (C) 2025 Gravitational, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package awsconfigfile
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/gravitational/trace"
+	ini "gopkg.in/ini.v1"
+)
+
+// AWSConfigFilePath returns the path to the AWS configuration file.
+func AWSConfigFilePath() (string, error) {
+	// See https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html
+	if location := os.Getenv("AWS_CONFIG_FILE"); location != "" {
+		return location, nil
+	}
+
+	homedir, err := os.UserHomeDir()
+	if err != nil {
+		return "", trace.Wrap(err)
+	}
+
+	return filepath.Join(homedir, ".aws", "config"), nil
+}
+
+// SetDefaultProfileCredentialProcess sets the credential_process for the default profile.
+// File is created if it does not exist.
+func SetDefaultProfileCredentialProcess(configFilePath string, credentialProcess string) error {
+	sectionName := "default"
```

Comment:
```suggestion
	const sectionName = "default"
```

I'm not a Go expert though, I'm not sure if the compiler is smart enough to do this optimization for us.

Reply:
Performance wise I don't think it matters, but this improves readability. Thank you